src/api/db/migrate/20180906142702_create_status_reports.rb

```diff
@@ -0,0 +1,16 @@
+class CreateStatusReports < ActiveRecord::Migration[5.2]
+  def change
+    create_table :status_reports, id: :integer, options: 'ROW_FORMAT=DYNAMIC' do |t|
+      t.string :uuid
+      t.belongs_to :checkable, polymorphic: { limit: 191 }, type: :integer
```

Comment:
What is the limit for?

Reply:
What @coolo comment?

Reply:
But if I get this right, you remove it in a later migration anyway - so just drop all of it? :)

Reply:
No we do not drop it the migration. This is what is going to happen.

Reply:
the 3 migrations are just within this PR, this is just squashing them.

Reply:
Yeah the migration test in the package fails without it as it's DB version dependent. :angel:  I :heart: this one...